movie/src/components/Card/index.jsx

```diff
@@ -33,7 +40,18 @@ const Card = ({ movies }) => {
     setCardId(0);
   };
 
-  const isCardClicked = (idCard, idClick) =>
+  // use on style.
+  const setCircleColor = (voteAverage) => {
```

Comment:
try not to have a function name start with the word "set" because might make other people confused with `setState`